docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -119,7 +144,7 @@ apiVersion: eksctl.io/v1alpha5
 metadata:
   name: ${CLUSTER_NAME:-camunda-cluster} # e.g. camunda-cluster
   region: ${REGION:-eu-central-1} # e.g. eu-central-1
-  version: "1.30"
+  version: "1.31"
```

Comment:
> Error: invalid version, supported values: 1.23, 1.24, 1.25, 1.26, 1.27, 1.28, 1.29, 1.30

Reply:
TIL, didn't think it was dependent on the eksctl verison.
I was on 0.190, my bad.